test/integration/query-interface/changeColumn.test.js

```diff
@@ -9,12 +9,9 @@ const { DataTypes } = require('@sequelize/core');
 const dialect = Support.getTestDialect();
 
 describe(Support.getTestDialectTeaser('QueryInterface'), () => {
-  beforeEach(function () {
+  beforeEach(async function () {
     this.sequelize.options.quoteIdenifiers = true;
     this.queryInterface = this.sequelize.getQueryInterface();
-  });
-
-  afterEach(async function () {
```

Comment:
What happens after the last test of this suite? Will the test schemas still be dropped? Or could this cause a flaky test as well?

Reply:
`beforeEach` from `integration/support.js` drops all schemas

Reply:
Ah yes